packages/web/src/app/doc-pages/components/context-menu-doc/context-menu-data.ts

```diff
@@ -37,6 +37,20 @@ const contextMenuData: ComponentData<BaseContextMenuProps> = {
         display = "block"; 
         display = "flex";`,
     },
+    anchorPosition: {
+      type: '',
+      description: 'Can be used to manually override the position of the context menu.',
+      children: {
+        top: {
+          type: 'number',
+          description: 'The top position of the context menu.',
```

Comment:
```suggestion
          description: 'The top position of the context menu in pixels.',
```
I wonder if we should specify that this number is pixels 🤔